src/cloud/components/onboarding/EngagementLink.tsx

```diff
@@ -19,12 +19,12 @@ const EngagementLink: FC = () => {
   const me = useSelector((state: AppState) => state.me)
 
   useEffect(() => {
-    if (isFlagEnabled('newUsageAPI')) {
-      handleGetUserStatus().then(({usageDataStates}) =>
+    if (isFlagEnabled('newUsageAPI') && org?.id) {
+      handleGetUserStatus(org.id).then(({usageDataStates}) =>
         setDataStates(usageDataStates)
       )
     }
-  }, [])
+  }, [org?.id])
```

Comment:
Will using the orgID here cause it to only update the usage state once per login or on full page reload? We may need to add another flag to recheck the usage state more frequently since it will change during the course of regular use. For instance, we could recheck on path change or something like that.